frontend/index.html

```diff
@@ -2,15 +2,35 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <title>The Hub | India Ultimate</title>
+  
+    <!-- Primary Meta Tags -->
+    <meta name="title" content="The Hub | India Ultimate" />
+    <meta name="description" content="India Ultimate's membership and tournament management portal" />
+  
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://hub.indiaultimate.org/" />
+    <meta property="og:title" content="The Hub | India Ultimate" />
+    <meta property="og:description" content="India Ultimate's membership and tournament management portal" />
+    <meta property="og:image" content="https://metatags.io/images/meta-tags.png" />
```

Comment:
will update this url after adding the banner in code right?

Reply:
yes